common/constants.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+const (
+	// InstanceID instance ID key for logging details
+	InstanceID = "instanceID"
```

Comment:
Sorry, what I meant was the values (and saying casing was misleading) - they're all for logging details, but some are `instanceID` and some are `dimension_id`, so whether we want the K/V pairs in logs to follow a predictable format